saleor/webhook/payloads.py

```diff
@@ -550,14 +553,18 @@ def generate_checkout_payload(
     quantize_price_fields(checkout, checkout_price_fields, checkout.currency)
     user_fields = ("email", "first_name", "last_name")
 
-    lines_dict_data = serialize_checkout_lines(checkout)
+    lines_dict_data = serialize_checkout_lines(checkout, allow_replica=True)
```

Comment:
Are we sure about this one to use replica? This webhook is called on checkout mutations where checkout lines can be modified, so are we 100% sure that the replica is up-to-date at this point?

Reply:
That's the fattest, most expensive call (because of the amount of prefetches it does deeper in `fetch_checkout_lines`) but yep, unfortunately I see it may cause issues as deeper described [here](https://github.com/saleor/saleor/issues/14472). One of ACs on that is that `The calls of async webhook triggered by mutation have set allow_replica to False` and this is direct example. I'm closing this one.